src/Samples/Toolkit.SampleApp.WPF_Net5/Samples/OverviewMap/OverviewMapSample.xaml

```diff
@@ -0,0 +1,56 @@
+﻿<UserControl
+    x:Class="Esri.ArcGISRuntime.Toolkit.Samples.OverviewMap.OverviewMapSample"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:toolkit="clr-namespace:Esri.ArcGISRuntime.Toolkit.UI.Controls;assembly=Esri.ArcGISRuntime.Toolkit"
+    d:DesignHeight="450"
+    d:DesignWidth="800"
+    mc:Ignorable="d">
+    <Grid>
+        <esri:MapView x:Name="MyMapView" />
+        <esri:SceneView x:Name="MySceneView" Visibility="Collapsed" />
+        <toolkit:OverviewMap
+            x:Name="SampleOverview"
+            Width="100"
+            Height="100"
+            Margin="8"
+            HorizontalAlignment="Right"
+            VerticalAlignment="Top" />
```

Comment:
I see you define 100x100 and upper-right a lot, and these are defaults. Do we really need them here? Since this is mainly a functional test app this also helps validating those defaults work.